snapshots/health/2024-04-12/polio_free_countries.csv.dvc

```diff
@@ -0,0 +1,26 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Polio-Free Countries - GPEI
+    date_published: "2016"
+
```

Comment:
You can maybe have a description for this snapshot, as the one in the webpage:
```
Eradicating polio has been a worldwide effort over many years. *This table shows the last recorded case of indigenous wild poliovirus (WPV) by country. As data methodology and quality varied widely across regions and countries in earlier years, this table is based on the best-available sources for the years before 2000.*

Until polio is completely eradicated, all countries remain at risk of imported WPV, as has happened in a number of countries.
```

Or just the bit between the asteriks

Reply:
Good idea!